workflows/validate.py

```diff
@@ -0,0 +1,78 @@
+"""
+Runs the json validator on a json file and raises an exception if there is an error
+Intended to be used as a hook for repositories which are developing recipes
+
+Example of how this could be used in a .pre-commit-config.yaml file:
+```
+# Validate json recipes
+- repo: https://github.com/DiamondLightSource/python-workflows
+  hooks:
+  - name: Recipe validation
+    id: recipe-validation
+    language: python
+    files: ^recipes/
+    entry: workflows-recipe-validate
+```
+"""
+
+import logging
+import workflows.recipe
+import workflows
+import sys
+import json
+import argparse
```

Comment:
Sort imports.
* add a `from __future__ import absolute_import, division, print_function`, that one goes on top, followed by an empty line, then
* all python system packages (here: anything but workflows*), alphabetically sorted, another empty line
* anything else, alphabetically sorted

It makes things much easier to read.

Reply:
Done